public/i18n/en/translation.json

```diff
@@ -660,7 +660,8 @@
       "no_devices_available": "No devices available",
       "custom_zone": "Custom",
       "cannot_remove_device_configuration": "Cannot remove device configuration",
-      "hotspot_network": "Network"
+      "hotspot_network": "Network",
+      "advanced_settings": "Advanced settings"
```

Comment:
We can use `common.advanced_settings`

Reply:
Whoops I forgot about it, should be fixed now